vsintegration/ItemTemplates/AppConfig/source.extension.vsixmanifest

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
+  <Metadata>
+    <Identity Id="Microsoft.VisualFSharp.ItemTemplates.AppConfig" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
+    <DisplayName>F# .NETCore Project Templates</DisplayName>
+    <Description>F# project templates for .NETCore.</Description>
+  </Metadata>
+  <Installation SystemComponent="true" Experimental="true">
```

Comment:
>SystemComponent="true" [](start = 16, length = 22)

Remove `SystemComponent="true"`. This will be filled in by Arcade when Willow component is generated.